Automattic/vip-go-mu-plugins
# Pull EP

## Description
See https://github.com/Automattic/ElasticPress/pull/87 for details


## Changelog Description

### Plugin Updated: Search

Fixing an issue in ElasticPress facet widget that could lead to some terms being ignored.
 
## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Implement `ep_facet_include_taxonomies` to enable taxonomies to be used for the facet
1. Add the facet
1. See results
